setup.py

```diff
@@ -67,13 +67,13 @@
             "pytest-cov>=3.0.0",
             "pytest-notebook>=0.6.1",
             "pytest-xdist>=2.4.0",
+            "opencv-contrib-python-headless==4.1.2.30",
             "ray[rllib]==1.0.1.post1",  # We use Ray for our multiprocessing needs
             "tensorflow>=2.4.0",  # For rllib tests
         ],
         "train": [
-            "ray[rllib]==1.0.1.post1",  # We use Ray for our multiprocessing needs
-            # XXX: TF requires specific version of scipy
-            "scipy==1.4.1",
+            "opencv-contrib-python-headless==4.1.2.30",
```

Comment:
This seem to conflict with the changes in PR #1185.

![image](https://user-images.githubusercontent.com/47334253/157926306-5972b4e2-e2f9-4d21-98e0-2df572efac8d.png)

Reply:
It will, I will merge master back into develop after this is merged in and undo these changes.